docs-kits/kits/behaviour-twin-kit/development-view/guidance-and-resources.md

```diff
@@ -0,0 +1,37 @@
+﻿---
+id: guidance-and-resources
+title: Guidance And Resources
+description: Behaviour Twin KIT
+---
+
+<div style={{display:'block'}}>
+  <div style={{display:'inline-block', verticalAlign:'top'}}>
+
+![Behaviour Twin KIT banner](@site/static/img/kit-icons/behaviour-twin-kit-icon-mini.svg)
+
+  </div>
+  <div style={{display:'inline-block', fontSize:17, color:'rgb(255,166,1)', marginLeft:7, verticalAlign:'top', paddingTop:6}}>
+Behaviour Twin KIT
+  </div>
+</div>
+
+## KNOWLEDGE AGENT
+
+The Behaviour Twin KIT builds up on top of the [Agents KIT](../../knowledge-agents/adoption-view/intro). Therefore, at least a basic understanding of the following semantic web technologies is mandatory:
+
+- SPARQL 1.1 Query Language, see [SPARQL on w3.org ![(external link)](../assets/external-link.svg)](https://www.w3.org/TR/sparql11-query/)
+- Resource Description Framework (RDF), see [RDF on w3.org ![(external link)](../assets/external-link.svg)](https://www.w3.org/RDF/)
+- RDF 1.1 Turtle, see [Turtle on w3.org ![(external link)](../assets/external-link.svg)](https://www.w3.org/TR/turtle/)
+- Web Ontology Language (OWL), see [OWL on w3.org ![(external link)](../assets/external-link.svg)](https://www.w3.org/OWL/)
+- Shapes Constraint Language (SHACL), see [SHACL on w3.org ![(external link)](../assets/external-link.svg)](https://www.w3.org/TR/shacl/)
+
+Catena-X-Standards regarding Knowledge Agents are:
+
+- [CX-0084 Federated Queries in Dataspaces (V1.2.0) ![(external link)](../assets/external-link.svg)](https://catena-x.net/de/standard-library/Update_Mai_2024/CX-0084-FederatedQueriesInDataSpaces-v1.2.0.pdf)
+- [CX-0067 Ontology Models to Realize Federated Queries in Catena-X (V1.1.0) ![(external link)](../assets/external-link.svg)](https://catena-x.net/de/standard-library/Update_Mai_2024/CX-0067-OntologyModelsinCatenaX-v1.1.0.pdf)
```

Comment:
As the links to the PDF-files won't work after the publication of R24.05, I suggest to change the link to: https://catena-x.net/de/standard-library